server/plugin/constants.go

```diff
@@ -23,6 +23,7 @@ const (
 	PathOpenDialog                 = "/api/v4/actions/dialogs/open"
 	PathSetDateTimeDialog          = "/date_time"
 	PathSetDateTime                = "/selected_date_time"
+	PathToSkip                     = "/skip"
```

Comment:
It should be similar to above paths as well like on line 23 variable name is PathOpenDialog not PathToOpenDialog.
```suggestion
	PathSkip                     = "/skip"
```